Graph types as direct PR (#1406)
neo4j/docs-cypher: modules/ROOT/pages/schema/graph-types/index.adoc

:description: Information about setting an open schema in Neo4j using graph types.

> :page-role: cypher-25-only new-neo4j-2025.xx enterprise-edition

Reviewer: ```suggestion
:page-role: cypher-25-only new-neo4j-2026.02 enterprise-edition
```

= Graph types

.Preview Feature
[IMPORTANT]
====
The *graph types feature is offered AS-IS* as described in your agreement with Neo4j and should only be used for internal development purposes.

When this feature becomes generally available, you will need to upgrade to the latest Neo4j version (which may require downtime) to use the feature for non-development purposes.

*Enabling the preview feature (internal parameter):* +
In Aura, graph types are enabled by default.

In self-managed Neo4j, graph types are disabled by default.

> Use the internal setting `internal.dbms.graph_types=true` to enable it.

Author: ```suggestion
Graph types are enabled by default, both in Aura and self-managed Neo4j.

Use the internal setting `internal.dbms.graph_types=false` if you wish to disable it for self-managed Neo4j.
```

> Use the internal setting `internal.dbms.graph_types.enabled=true` to enable it.

Author: ```suggestion
Use the internal setting `internal.dbms.graph_types=true` to enable it.
```

:page-role: cypher-25-only new-neo4j-2025.xx enterprise-edition
= Graph types

.Preview Feature
[IMPORTANT]
====
The *graph types feature is offered AS-IS* as described in your agreement with Neo4j and should only be used for internal development purposes.

When this feature becomes generally available, you will need to upgrade to the latest Neo4j version (which may require downtime) to use the feature for non-development purposes.

*Enabling the preview feature (internal parameter):* +
In Aura, graph types are enabled by default.

In self-managed Neo4j, graph types are disabled by default.

> Use the internal setting `internal.dbms.graph_types.enabled=true` to enable it.

Reviewer: Isn't this just `internal.dbms.graph_types=true`? 🤔

Author: that could very well be the case :) - this was from brain brump after seeing it briefly in the document, and it's something yet to be double-checked
